pinning.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+// Pinning configures the pinning services.
+type Pinning struct {
+	// Services lists the pinning services
+	Services []PinningServices
+}
+
+type PinningServices struct {
+	Name        string
+	ApiEndpoint string
+	ApiKey      string
```

Comment:
I would also add `IconURL string` here

Reply:
What if UI checks for  `/icon` at endpoint and if it returns 404 use a placeholder image instead?
That would not require any additional configuration, similar to `/favicon.ico` on the web.

OR we could have it as a field, but `IconCID` instead of `URL`.

Reply:
After thinking about this, I don't think we should store IconURL in daemon config.

WebUI could simply keep a mapping between `new URL(ApiEndpoint).origin` and icons for initial partners, and if there is no predefined icon for a Custom service, a randomized identicon (https://github.com/ipfs-shipyard/ipfs-webui/issues/935)  could be generated based on `Name+ApiEndpoint`